Translation Portuguese Marketing (#1125)
ropensci/roweb3: content/blog/2024-03-07-package-marketing/index.pt.md

---
title: Ideias de marketing para seu pacote
author:
- Yanina Bellini Saibene
- Maëlle Salmon
editor:
- Steffi LaZerte
date: '2024-03-07'
slug: package-marketing
tags:
- community
- maintenance
- package-development

> description: Agora que você criou seu pacote, apresentá-lo ao mundo é uma etapa crucial

Reviewer: ```suggestion
description: Agora que você criou o seu pacote, apresentá-lo ao mundo é uma etapa crucial
```

  para ganhar visibilidade e atrair usuários.  Nesta publicação do blog, sugerimos
  uma série de atividades e ferramentas para você divulgar seu pacote.
params:
  doi: 10.59350/04b0t-fv224
---

> Agora que você criou seu pacote, apresentá-lo ao mundo é uma etapa crucial para **ganhar visibilidade e atrair usuários**.

Reviewer: ```suggestion
Agora que você criou o seu pacote, apresentá-lo ao mundo é uma etapa crucial para **ganhar visibilidade e atrair usuários(as)**.
```

O marketing eficaz do seu pacote contribui para alcançar as pessoas que ele pode apoiar, encontrando usuários para ajudá-lo a manter e aprimorar o pacote e permitindo que você saiba como as pessoas o utilizam.

Nesta publicação do blog, sugerimos uma série de atividades e ferramentas para você divulgar seu pacote.

## Publicar

Crie um documento abrangente [LEIAME](https://blog.r-hub.io/2019/12/03/readmes/) que explique claramente o que o seu pacote faz, como instalá-lo e como os usuários podem começar a usá-lo. Inclua exemplos ou casos de uso e (links para) qualquer informação relevante que possa ajudar os usuários a entender como o seu pacote pode ajudá-los.

Se você usa o GitHub, pode [fixar o repositório do pacote em seu perfil](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/pinning-items-to-your-profile) para que outras pessoas possam encontrá-lo rapidamente.

> [Crie um universo no R-universe](/blog/2021/06/22/setup-runiverse/) que ajudará os usuários simplificando a instalação do seu pacote e que fornecerá estatísticas e informações úteis sobre ele. O R-Universe também é uma forma de mostrar a coleção de pacotes que você possui. Você também pode [publicá-la no CRAN](https://cran.r-project.org) que é outra forma de simplificar a instalação para seus possíveis usuários.

Reviewer: @yabellini @maelle, the link appears to be broken </blog/2021/06/22/setup-runiverse/>.

Por fim, considere a possibilidade de enviar seu pacote para o nosso [processo de revisão por pares](/software-review/) se você estiver no escopo. Todas essas etapas ajudarão você a ser descoberto e também a cumprir os padrões de qualidade. Se isso for de interesse para você, geralmente recomendamos [que você envie para revisão antes de enviar para o CRAN](https://devguide.ropensci.org/policies.html#publishing-in-other-venues) pois os comentários de revisão podem ajudar você a enviar para o CRAN.

## Promover

> Agora que o seu pacote está publicado e pronto para ser usado, aqui estão algumas sugestões de coisas que você pode fazer para ajudar a promover o seu pacote e criar uma base de usuários.

Reviewer: ```suggestion
Agora que o seu pacote está publicado e pronto para ser usado, aqui estão algumas sugestões de coisas que você pode fazer para ajudar a promover o seu pacote e criar uma base de usuários(as).
```

Aproveite plataformas como Mastodon, LinkedIn e fóruns específicos de R, como o [R-bloggers](https://www.r-bloggers.com) e [R Weekly](https://rweekly.org) para anunciar o lançamento do seu pacote. Você também pode usar sua própria mídia social ou blog para compartilhar atualizações, tutoriais, casos de uso, palestras e eventos futuros em que você estará disponível para interagir com os usuários.

> Se o seu pacote fizer parte do rOpenSci Suite, você poderá [escrever uma publicação no blog ou uma nota técnica](https://blogguide.ropensci.org/) para o nosso blog e nós compartilharemos em nosso [newsletter mensal](/news/) e em nossas mídias sociais.

Reviewer: ```suggestion
Se o seu pacote fizer parte da rOpenSci Suite, você poderá [escrever uma publicação ou uma nota técnica](https://blogguide.ropensci.org/) para o nosso blog e nós compartilharemos em nosso [newsletter mensal](/news/) e em nossas mídias sociais.
```

Para os pacotes rOpenSci, coletamos [casos de uso](/usecases) e os apresentamos no site, bem como [no pacote](/packages/). Também temos campanhas de mídia social, por exemplo ["Um pacote por dia"](https://hachyderm.io/@rOpenSci/tagged/APackageADay) no Mastodon, onde apresentamos um pacote de nossa suíte por dia. No LinkedIn, publicamos toda segunda-feira nosso ["Resumo semanal de pacotes"](https://www.linkedin.com/feed/update/urn:li:activity:7170464303186108416) com uma lista de cinco pacotes que pertencem à mesma categoria.

Se você gosta de dar palestras, pode falar em um grupo de usuários do R ou em um grupo de usuários do R. [Capítulo R-Ladies](https://www.meetup.com/pro/rladies/). Os organizadores convidam palestrantes, mas a maioria dos grupos está aberta a ofertas para dar uma palestra ou ministrar um workshop. Apresentar seu pacote em uma conferência específica do domínio é outra forma de alcançar os usuários e obter maior exposição. Há também conferências específicas do R, como a [useR!](https://events.linuxfoundation.org/user/), [LatinR](https://latin-r.com/), [dias de sábado](https://satrdays.org/) e [Posit::Conf](https://posit.co/conference/).

> rOpenSci tem duas horas de duração [sessões de co-working de duas horas todos os meses](/coworking/) Se você estiver interessado em organizar uma sessão de trabalho conjunto sobre um tópico relacionado ao seu pacote, entre em contato conosco.

Reviewer: ```suggestion
A rOpenSci tem duas horas de [sessões de trabalho em conjunto todos os meses](/coworking/). Se você tiver interesse em organizar uma sessão de trabalho conjunto sobre um tópico relacionado ao seu pacote, entre em contato conosco.
```

Se o seu pacote tiver um escopo amplo, você também pode propor uma [Chamada da comunidade rOpenSci](/commcalls/) sobre seu pacote. Por exemplo, uma chamada da comunidade sobre [alvos](/commcalls/jan2023-targets/) incluiu um painel com dois usuários mostrando como eles usam o pacote e o autor principal compartilhando novas percepções sobre o pacote. Como alternativa, considere a possibilidade de se reunir com outros autores de pacotes e propor uma chamada da comunidade relacionada ao domínio, abrangendo o uso de vários pacotes mais específicos (por exemplo, [Aplicativos de pesquisa das ferramentas de taxonomia e biodiversidade do rOpenSci](/commcalls/2019-03-27/)).

> Você pode verificar se o seu pacote se encaixa em um [Visualização de tarefas do CRAN](https://cran.r-project.org/web/views/) e propor a adição de seu pacote.

Reviewer: ```suggestion
Você pode verificar se o seu pacote se encaixa em uma [Visualização de tarefas do CRAN](https://cran.r-project.org/web/views/) e propor a adição dele.
```

Se o seu objetivo for encontrar ajuda para manter o seu pacote, rotule seus problemas como "Help Wanted" ou "Good First Issue" para orientar os colaboradores onde o suporte deles é necessário. [As edições do Help Wanted dos pacotes do pacote rOpenSci são apresentadas em nossa página da Web](/help-wanted/) compartilhadas nas mídias sociais e em nosso boletim informativo.  Também ajudamos os mantenedores a escrever uma nota técnica para a publicação em nosso blog com mais detalhes sobre o tipo de contribuição de que precisam em seus pacotes, seja ela [contribuições de código](/blog/2024/03/01/qualtrics-call-help/), [fechar problemas](/blog/2024/02/27/assertr-call-help/) ou [responder a perguntas](/blog/2024/02/29/targets-call-help/).

> A publicação de um artigo sobre o seu pacote é um aspecto importante para os desenvolvedores acadêmicos. Os periódicos gerais do R para publicação de pacotes do R são [The R Journal](https://journal.r-project.org) e o [Journal of Statistical Software](https://www.jstatsoft.org/index) mas lembre-se de que os periódicos específicos de um domínio também podem publicar pacotes R como ferramentas para o campo (por exemplo, [Methods in Ecology and Evolution](https://besjournals.onlinelibrary.wiley.com/journal/2041210x))

Reviewer: ```suggestion
A publicação de um artigo sobre o seu pacote é um aspecto importante para os(as) desenvolvedores(as) acadêmicos(as). Os periódicos gerais do R para publicação de pacotes do R são [The R Journal](https://journal.r-project.org) e o [Journal of Statistical Software](https://www.jstatsoft.org/index) mas lembre-se de que os periódicos específicos de uma área também podem publicar pacotes R como ferramentas para o campo (por exemplo, [Methods in Ecology and Evolution](https://besjournals.onlinelibrary.wiley.com/journal/2041210x))
```

> Os pacotes que são revisados por pares pela rOpenSci têm um caminho rápido para publicação na [Journal Open Source Software](https://joss.theoj.org).

Reviewer: ```suggestion
Os pacotes que são revisados por pares pela rOpenSci têm um caminho rápido para a publicação na [Journal Open Source Software](https://joss.theoj.org).
```

## Conclusão

> Como mantenedor de um pacote, há várias atividades que você pode implementar para divulgar seu pacote. Se o seu pacote fizer parte do pacote do rOpenSci, também há maneiras de oferecer suporte a você. Ao implementar algumas dessas opções, esperamos que você possa alcançar efetivamente os usuários que podem se beneficiar da funcionalidade do seu pacote.

Reviewer: ```suggestion
Como mantenedor(a) de um pacote, há várias atividades que você pode implementar para divulgá-lo. Se o seu pacote fizer parte dos pacotes da rOpenSci, também há maneiras de oferecer suporte a você. Ao implementar algumas dessas opções, esperamos que você possa alcançar efetivamente as pessoas que podem se beneficiar da funcionalidade do seu pacote.
```
